rootfs/registry/dockerclient.py

```diff
@@ -55,8 +54,8 @@ def publish_release(self, source, config, target, deis_registry):
     def build(self, source, config, repo, tag):
         """Add a "last-mile" layer of environment config to a Docker image for deis-registry."""
         check_blacklist(repo)
-        env = ' '.join("{}='{}'".format(
-            k, v.encode('unicode-escape').replace("'", "\\'")) for k, v in config.viewitems())
+        env = ' '.join("{}='{}'".format(k, v) for k, v in config.items())
```

Comment:
I recall it was some ugly json parsing error... Might wanna do a `git blame` on that and figure out what the history behind this ugly code fixes